dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpServiceContextImpl.java

```diff
@@ -3363,78 +3378,42 @@ final protected void sendFullOutgoing(WsByteBuffer[] wsbb) throws IOException {
             }
         }
 
-        DefaultHttpContent content;
         boolean shouldSkipWriteOnUpgrade = nettyResponse.status().equals(HttpResponseStatus.SWITCHING_PROTOCOLS)
                                            && !nettyContext.channel().attr(NettyHttpConstants.PROTOCOL).get().equals("HTTP2");
```

Comment:
Switch "HTTP2".equals( attribute) to protect NPE

Reply:
Created https://github.com/OpenLiberty/open-liberty/issues/32231 to follow up